tests/test_app.py

```diff
@@ -436,3 +439,18 @@ def test_warning_config_options_on_compile_command(snippetcompiler, tmpdir):
     assert "Compile done" in all_output
     assert f"Config directory {non_existing_config_dir} doesn't exist" not in all_output
     assert f"Config file {non_existing_config_file} doesn't exist" not in all_output
+
+
+@pytest.mark.parametrize(
+    "with_tty, regexes_required_lines, regexes_forbidden_lines",
+    [(False, [r"Current Inmanta version:"], []), (True, [r"Current Inmanta version:"], [])],
```

Comment:
The second and the third parameter are always the same, so it doesn't make sense to include it in the parameterization.

Reply:
Good point